src/mcdu-server/index.js

```diff
@@ -50,9 +50,11 @@ let selectedPrinter = null;
 // Parse command line args
 let printerName = null;
 let skipPrinter = false;
-let httpPort = 8125;
-let websocketPort = 8080;
+let httpPort = 8320;
```

Comment:
Changing the HTTP port will be quite annoying for existing users with bookmarks/homescreen links. Do we need to? Websocket port not such an issue as it's not really user visible.

Reply:
I agree. Support would be busy :)